### PR TITLE
creating/tracking: make sure we add the CSV

### PR DESCRIPTION
**Pull request by dunn**
Closes GH-76.

### CODE REVIEW

File: _episodes/03-changes.md [hunk 1]
```diff
@@ -407,23 +425,23 @@ $ git log
...
     readme: where are the files?
 
-commit 73f865ac9f0b3e790328d11ec6cd7653208a5a3e
+commit 9ab6e19c9872a23674772ea5a514f05a65ba5d9e
 Author: Catsy Cline <ccline@pawson.edu>
-Date:   Wed Oct 25 15:44:38 2017 -0700
+Date:   Thu Nov 2 12:03:47 2017 -0700
 
-    add notes on metadata
+    add notes on metadata, and the metadata
```
Comment (chrissyrissmeyer) [inline _episodes/03-changes.md]: might want to check the rest of the episodes for this commit message... i think it is used elsewhere as an example

Comment (chrissyrissmeyer) [inline _episodes/03-changes.md:444]: you might want to check the other episodes for this commit message... i think it is used in our examples elsewhere

Comment (chrissyrissmeyer) [reply _episodes/03-changes.md:444]: we could also tackle that as part of https://github.com/ucsdlib/git-novice/issues/59

Comment (dunn) [reply _episodes/03-changes.md:444]: I was thinking with the updates @mcritchlow is planning to do for Ignoring Things, we might have to re-do subsequent examples anyway.

Comment (chrissyrissmeyer) [reply _episodes/03-changes.md:444]: okay, sounds good

**Review by chrissyrissmeyer — APPROVED** [review]

**Pull request merged**